nss-day-cohort-60/national-parks-client-v2
# Jd photo migrations

# Description
Changed all instances of cloudinary image source to use local server image storage"

# Steps to test

1. Fetch the JD-photo-migrations branch and switch to it
2. Fetch the JD-photo-migration server branch and switch to it
3. Visit /home and verify park photos still display in carousel
4. Visit /parks/# and verify carousel, blog photos, campground photos, wildlife photos render correctly
5. Visit Blogs and verify correct images
6. Visit User Hub and verify carousel, blog images display correctly
7. Check network tab in dev tools and confirm no traffic with cloudinary - images should all come from some variation of localhost:8000/media/photos/{image file name}
